src/Voron/Data/Tables/Table.cs

```diff
@@ -2164,7 +2156,7 @@ public TableReport GetReport(bool includeDetails, StorageReportGenerator generat
         {
             generatorInstance ??= new StorageReportGenerator(_tx.LowLevelTransaction);
 
-            var overflowSize = _overflowPageCount * Constants.Storage.PageSize;
```

Comment:
Perhaps we want to create a computed property inside `TableSchemaStatsReference`?

```csharp
public long OverflowSize => OverflowPageCount * Constants.Storage.PageSize;
```

Reply:
There is just one call like that. Let's leave as is.